File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntPipelineVar.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Symbol;
import com.neome.java.api.meta.base.Types.MetaIdForm;
import com.neome.java.api.meta.base.Types.MetaIdPipelineVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntPipelineVar extends StudioBase
{
  @Nullable
  public MetaIdForm formId;

  public MetaIdPipelineVar metaId;

  public Symbol name;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntPipelineVar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdPipelineVar
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.Symbol

interface StudioEntPipelineVar : StudioBase
{
  val formId: MetaIdForm?
  val metaId: MetaIdPipelineVar
  val name: Symbol
}